packages/editor/src/components/ui/Formula/FormulaResult.tsx

```diff
@@ -10,13 +10,16 @@ import {
 import { FormulaValue, FORMULA_ICONS } from '.'
 import * as Root from './Formula.style'
 import { UseFormulaInput } from '../../blockViews/FormulaView'
+import { useFormulaI18n } from '../../../hooks/useFormulaI18n'
 
 export interface FormulaResultProps {
   variableT: VariableData | undefined
   meta: UseFormulaInput['meta']
 }
 
 export const FormulaResult: FC<FormulaResultProps> = ({ variableT, meta }) => {
+  const { t } = useFormulaI18n()
```

Comment:
`useFormulaI18n()` will suspend rendering while fetching i18n resources.
Be sure to wrap `<FormulaResult>` in a `<Suspense fallback={...}>` to properly handle loading state.
Otherwise the nearest `<Suspense>` (which maybe one of `<Suspense>` in `DocumentContentPage.tsx` or the one in `App.tsx`) will catch the suspense and show it fallback, which may not be what you expect (e.g. the most outer one in `App.tsx` will show a full screen loading animation during suspense)

Reply:
cc @aligo @Darmody @melchior-voidwolf in case you are not familiar with React <Suspense>